scripts/jenkins/Jenkinsfile

```diff
@@ -46,20 +46,20 @@ def buildClosures(arg) {
                 stage("build-${platform}") {
                     rebuild = detectChanges()
                     docker.build("${platform}-master-test","${rebuild} -f ./scripts/docker/build-${platform}/Dockerfile.deps ./scripts/docker/build-${platform}").inside {
-                        checkout([$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'CleanBeforeCheckout'],[$class: 'RelativeTargetDirectory', relativeTargetDir: "${platform}/build"]], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/freeradius/freeradius-server']]])
+                        checkout([$class: 'GitSCM', branches: scm.branches, userRemoteConfigs: scm.userRemoteConfigs, extensions: [[$class: 'CleanBeforeCheckout'],[$class: 'RelativeTargetDirectory', relativeTargetDir: "${platform}/build"],[$class: 'CloneOption', depth: 0, noTags: false, reference: '', shallow: false]], submoduleCfg: []])
                         sh "cat /etc/os-release || cat /etc/redhat-release"
+                        def commit_num = readFile("./build-number").trim()
```

Comment:
Hm, what's the reason for doing it this way instead of running the git describe command directly in the Jenkins file

Reply:
I don't see a problem with it... just interested.  Anyway, will merge this now.

Reply:
Basically I didn't want to run that command more than once in the Jenkinsfile, to avoid future problems, and I had to do it outside of the docker image and the parallel builds to create the file that the updaterepo job reads.  I was hoping to be able to pass that in to the docker images as a variable, but that proved impossible. I had to write that file anyway, so I just had the builds read it, rather than run the whole big command again.

Reply:
OK, fair enough.  The command is still accessible within the Jenkins file which is the main thing.

Reply:
I did a lot of work trying to get it so that things were only set in one place if possible, just to reduce the risk of copypasta errors later on.